examples/E2E-RAG-App/README.md

```diff
@@ -0,0 +1,63 @@
+## E2E-RAG-App
+
+This is an End to End RAG App leveraging llama-stack that handles the logic for ingesting documents, storing them in a vector db and providing an inference interface.
+
+We share the details of how to run first and then an outline of how it works:
+
+### Prerequisite:
+
+Install docker: Check [this doc for Mac](https://docs.docker.com/desktop/setup/install/mac-install/), [this doc for Windows](https://docs.docker.com/desktop/setup/install/windows-install/) and this [instruction for Linux](https://docs.docker.com/engine/install/).
+
+For Mac and Windows users, you need to start the Docker app manually after installation.
+
+### How to run the pipeline:
+
+![RAG_workflow](./RAG_workflow.jpg)
+
+The above is the workflow diagram for this RAG app. To run the app, please read the following instructions:
+
+1. We have main config `RAG_service.json` inside of the docker folder, please change `model_name` and `document_path` accordingly, for example:
+
+```yaml
+{
+  "model_name": "llama3.2:1b-instruct-fp16",
+  "document_path": "${HOME}/work/llama-stack-apps/examples/E2E-RAG-App/example_data"
+}
+```
+
+2. Inside of docker folder, `run_RAG.sh` is the main script that can create `.env` file for compose.yaml and then actually start the `docker compose` process to launch all the pipelines in our dockers. `compose.yaml` is the main docker yaml that specifies all the mount option and docker configs, change the mounts if needed.
```

Comment:
these details are unnecessary I think. these scripts are very simple and self-documenting in a way.

Reply:
Sure.. I will make a short version